### ISSUE TITLE
[request] Code input

### ISSUE THREAD

**Issue by sinus-x** [benawad/vsinder-app#6]
I think it would be nice if the font in the Code input was monospace. It is... code, after all.

**Comment by benawad** [comment]
Only supporting fonts that Carbon has atm

**Comment by logangeorge01** [comment]
He means the actual text input in the app should be monospace, nothing to do with carbon.

**Comment by benawad** [comment]
Oh, I think it uses whatever your VSCode font is?

**Comment by benawad** [comment]
Oh, I could load a custom font, which one do you want?

**Comment by logangeorge01** [comment]
My vscode default settings have Menlo as the monospace font so I think that would be good if it's free.

**Comment by sinus-x** [comment]
Yes, I meant the font of the input field itself, I should have been more clear.

And as far as I'm concerned, I don't care much _which_ monospace font will be used. I'm used to the Ubuntu Mono, but any font will do, as long as it is non-proportional and allows me to see how the snippet will look like rendered out as an image.
